bundle/regal/ast.rego

```diff
@@ -402,6 +402,31 @@ is_chained_rule_body(rule, lines) if {
 	startswith(col_text, "{")
 }
 
+imports_keyword(imports, keyword) if {
+	some imp in imports
+
+	_has_keyword(imp, keyword)
+}
```

Comment:
[style] I wonder if Regal would flag this 🙃 

```suggestion
imports_keyword(imports, keyword) if _has_keyword(imports[_], keyword)
```

Reply:
It would, I think, but see #425 suggests allowing more exceptions to that.